### PR TITLE
Temporary removal of fleshshot due to server crashes.

### PR DESCRIPTION
**Pull request by N3X15**
On October 2nd, we experienced a recurrance of the server crashes that have hit us in the past with regards to fleshshots.  I spent an hour or so trying to reproduce the issue by using the hookshot, and then by becoming a borer and infesting a human and evolving the fleshshot.  I was unable to reproduce the problem, despite clicking on nearly every object on screen.

This issue has come and gone for months, and despite several attempts at fixing it, it has persisted.  Due to the severity of the problem, I have decided to remove it, both to prevent a recurrence, and to spur people into helping to debug and fix it.

And yes, I realize this will not make me popular but that seems to be my niche in life.

# Changelog
:cl:
* rscdel: Temporarily removed fleshshot from borer evolution tree while a severe server crash issue is fixed.

/:cl:

# Issues
* #16151 
* #15092

### CODE REVIEW

**Comment by N3X15** [comment]
The biggest problems with hookshots and fleshshots that I see, at the present:

* The design is flawed.  As a design oversight, hookshot chains can become attached to themselves, which can cause crashes in the server.  I don't know if all crashes are the result of this issue, but it's certainly not helped.
* The code for fleshshots is largely copypasta of hookshots, which means that issues fixed in hookshots are not transmitted via inheritance to fleshshots.  Fleshshots should make better use of inheritance and avoid copypasta where possible.
* Another design flaw is the fact that multiple fleshshot-capable borers can fire at once, resulting in both borers pulling the host in two directions at once (which I am told by Oliver is a crash condition) and chains colliding, which I'm certain is also a crash condition.

tl;dr they're a goddamn mess.

**Comment by Hugh-G-Rection** [comment]
Since this is happening regardless, I should also mention that arm borer can be used as a ghetto bound-object. Certain actions like the host attempting to throw something, drop something or being pushed over while the borer is swinging it can result in ghosthands, and the next time the borer hits Q the item will re-appear at the hosts feet.

I hope this results in this fun tool being fixed and made better, and not removed indefinitely.

**Comment by Shadowmech88** [comment]
I will attempt to fix the issues.
